### ISSUE TITLE
Improve Note Positioning Using Tracks

### ISSUE THREAD

**Issue by winder** [pianobooster/PianoBooster#255]
A lot of MIDI files seem to be organized by tracks to indicate the left and right hand. This can also be used as an easy way to decide where to put notes which land on/around middle C.

I've been working on Bach's Prelude and this trick greatly improves the note placement when compared to actual sheet music.

These screenshots are from a pet project I've been working on for the past couple weeks. I've never worked with MIDI before this week, so there are probably additional improvements to make.

Before - Everything below middle C is placed on the bass clef. Middle C and above on the treble clef.
![before](https://user-images.githubusercontent.com/125509/93669365-bdb8d480-fa61-11ea-8b96-b6e1454cfb19.png)

After - Everything from track 1 on the treble clef, everything from track 2 on the bass clef.
![after](https://user-images.githubusercontent.com/125509/93669366-c01b2e80-fa61-11ea-9d69-40375140988f.png)

The same MIDI file loaded into piano booster 0.7.3-PREVIEW
![pb](https://user-images.githubusercontent.com/125509/93711027-c3bdbc80-fb18-11ea-916e-09337f933bd1.png)

The MIDI file comes from this page: https://musescore.com/user/101554/scores/117279

**Comment by louis-barman** [comment]
This has been already fixed in #228. Sorry for the very slow delay in getting version 0.7.4 released. But the code is available now in the develop branch. Are you using  Linux or windows or mac?? If you are on  Linux it is not too hard to compile from source.

You first need to install the following packages   
```
build-essential, cmake, pkg-config, libftgl-dev, qtbase5-dev, qttools5-dev, librtmidi-dev , fluid-soundfont-gm, libfluidsynth-dev
``` 

Then create a directory called `build`  and then in that directory type `cmake .. -DWITH_INTERNAL_FLUIDSYNTH=ON` and then type `make`

**Comment by winder** [comment]
Amazing, I didn't notice that you had an edge branch! I am on linux so I can give that a try. Thanks for your efforts.
